### DISCUSSION TITLE
No braking at the end of the movement block on spike prime

### DISCUSSION THREAD

**Discussion by wertwq47** [pybricks/support#1919] [Q&A]
I created my blocks on spike prime. Back when I used ev3, my motor block had PID correction, acceleration and deceleration all correct. I tried to bring all the concepts and I succeeded. The problem is that when I tried to make me choose whether or not the robot locked the wheels at the end, it didn't work. For example, in the past, I would put a movement block to go straight with 80 maximum speed and in the next block I wanted it to continue at 80, but to make a curve, now it simply seems impossible.

When I tested the first movement block with the floating pattern and the next movement block with the same speed as the first one, it stopped anyway.

If the first movement block is floating by default and there is nothing after it, it never stops, it just keeps walking in a straight line forever.

 I don't understand how this Pybricks site works for my program and looking on the site I didn't t find anything about my program. I found this site randomly, but if anyone can help me I would be very grateful

**Comment by BertLindeman** [comment]
Hello,

Could you show the program you are having a problem with?
That can help us help you.


I understand you have a block program.
On the code.pybricks.com website you can make a picture of your program and copy / paste that here as a reply.

This is the "Make a picture of my program" button:
![image](https://github.com/user-attachments/assets/c0773b34-064c-4221-a8fd-4276c823c56c)
 
Bert

**Comment by laurensvalk** [comment]
To continue moving after one block (and smoothly continue with the next), choose 'continue moving'.

As the name implies, brake will brake. Like in a car, braking will slow you down regardless of which way you are going, but not reverse to correct overshoot.

So if you want to hold position, choose 'hold'. (They used to call this "brake" in the EV3 app, but this is a misnomer as explained above.)

**Comment by wertwq47** [comment]
![Move Block 1](https://github.com/user-attachments/assets/fd860564-b552-4feb-bb24-cbda8e1dfcba)
![Move Block 2](https://github.com/user-attachments/assets/b30c8011-484a-4b29-99bb-9034558111b8)
![Move Block 3](https://github.com/user-attachments/assets/b8081e6d-733b-416d-a755-f97724473252)

**Reply by wertwq47** [reply]
The specific part that needs to be braked is in this last print
